src/Service/Program.cs

```diff
@@ -195,6 +195,21 @@ public static ILoggerFactory GetLoggerFactoryForLogLevel(LogLevel logLevel, Tele
                         });
                     }
 
+                    if (Startup.AzureLogAnalyticsOptions.Enabled && !string.IsNullOrWhiteSpace(Startup.AzureLogAnalyticsOptions.Auth?.CustomTableName) &&
```

Comment:
Can you please verify logs continue to flow to AppInsights and Open Telemetry in addition to LogAnalytics?